core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -68,13 +67,20 @@ public function getResponse()
 	 */
 	protected function compile()
 	{
-		$this->ua = '';
+		$classes = array();
 
 		if (Config::get('fullscreen'))
 		{
-			$this->ua .= ' fullscreen';
+			$classes[] = 'fullscreen';
 		}
 
+		if (Input::get('popup'))
+		{
+			$classes[] = 'popup';
```

Comment:
This seems to be new? Wouldn't that add duplicate `popup` class for templates that already include it?

Reply:
The links to these pages do not include `?popup=1`, therefore no. However, we might consider changing this and removing the fixed `popup` class to make it more consistent?

Reply:
I was referring to the `be_ popup` template, which already includes this class.